### ISSUE TITLE
ONNX model performs wrongly and Normalization for Image

### ISSUE THREAD

**Issue by pthinh14** [dotnet/machinelearning#5337]
### System information

- **OS version/distro**: Window 
- **.NET Version (eg., dotnet --info)**: .NET Core 3.1

### Issue

- **What did you do?**
 I use pre-trained ONNX in my application
- **What happened?** 
The ONNX model performs worse in ML.NET than in other platforms (pytorch, tensorflow) in term of accuracy. They give the same results for some images while some are not. I think the problem might be on the preprocessing stage. In Pytorch, I have normalization step while I cannot produce the same method in ML.NET.
I tried NormalizeMeanVariance and NormalizeMinMax but the result is all 0 so I wonder if they are applicable for image.
- **What did you expect?** 
Normalization for image or allowing custom preprocessing transform. I would love to have sample for this problem. Actually it would be nice to have document about how to map Tensorflow, Pytorch functions to ML.NET functions.

### Source code / logs
ML.NET code which can produce some good results:
```
var pipeline = mlContext.Transforms
                .LoadImages(
                    outputColumnName: "image_object",
                    imageFolder: null,
                    inputColumnName: nameof(ImageData.ImagePath))
            .Append(mlContext.Transforms.ResizeImages(
                outputColumnName: "image_object_resized",
                imageWidth: ImageSettings.imageWidth,
                imageHeight: ImageSettings.imageHeight,
                inputColumnName: "image_object"
               ))
            .Append(mlContext.Transforms.ExtractPixels(
                outputColumnName: "image_object_scale",
                inputColumnName: "image_object_resized",
                offsetImage:117,
                scaleImage:1/255f))
.Append(mlContext.Transforms.ApplyOnnxModel(modelFile: _modelFilePath, outputColumnName: ModelSettings.outputTensorName, inputColumnName:  ModelSettings.inputTensorName));
```

**ML.NET normalize code:**
```
var pipeline = mlContext.Transforms
                .LoadImages(
                    outputColumnName: "image_object",
                    imageFolder: null,
                    inputColumnName: nameof(ImageData.ImagePath))
            .Append(mlContext.Transforms.ResizeImages(
                outputColumnName: "image_object_resized",
                imageWidth: ImageSettings.imageWidth,
                imageHeight: ImageSettings.imageHeight,
                inputColumnName: "image_object"
               ))
            .Append(mlContext.Transforms.ExtractPixels(
                outputColumnName: "image_object_scale",
                inputColumnName: "image_object_resized"))
            .Append(mlContext.Transforms.NormalizeLogMeanVariance(inputColumnName: "image_object_scale", outputColumnName: "image_normalized", fixZero:true));

            var normalizeTransform = pipeline.Fit(dataView);
            var transformedData = normalizeTransform.Transform(dataView);
            var column = transformedData.GetColumn<float[]>("image_normalized").ToArray();
            foreach (var row in column)
                    Console.WriteLine(string.Join(", ", row.Select(x => x.ToString())));
```

**Pytorch transformation:**
```data_transforms = transforms.Compose([
transforms.Resize(224),
transforms.ToTensor(),
transforms.Normalize([0.5,0.5,0.5], [0.5,0.5,0.5])
])```

**Comment by harishsk** [comment]
Hi @pthinh14 We are missing a transform in ML.NET to normalize images correctly. We are looking to add it in one of the upcoming release. Until then, you can use a [CustomMapping ](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.custommappingcatalog.custommapping?view=ml-dotnet )transform to do this. Please refer to [this example](https://github.com/dotnet/machinelearning-samples/blob/master/samples/modelbuilder/ImageClassification_Azure_LandUse/LandUseML.Model/NormalizeMapping.cs) on how to go about it.
